test/ir/pir/translator/test_nop_translator.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import test_op_translator
+
+import paddle
+from paddle.base.layer_helper import LayerHelper
+
+
+class TestNopTranslator(test_op_translator.TestOpTranslator):
+    def append_op(self):
+        self.op_type = "nop"
+        x = paddle.ones(shape=(1), dtype='float32')
```

Comment:
```suggestion
        x = paddle.ones(shape=(1,), dtype='float32')
```
现在报错处的代码如下：
```
def check_shape(shape):
    """
    Check shape type and shape elements type before passing it to fill_constant
    """
    if isinstance(shape, Variable):
        check_dtype(shape.dtype, 'shape', ['int32', 'int64'], 'fill_constant')
    else:
        for ele in shape:
```
从报错提示也可以看出这里我们传入的shape是一个int而非list。

这是因为，在python中，元组要这样写：
```
>>> type((1))
<class 'int'>
>>> type((1,))
<class 'tuple'>
```
可以改下看看。下面的out也要做同样的修改。

Reply:
多谢，重新触发ci可以通过了~